Dockerfile

```diff
@@ -37,7 +37,14 @@ WORKDIR /app
 
 COPY ./src/frontend /app/
 
-RUN yarn install --frozen-lockfile && \
+# We first fake an empty translation file to build the front a first time.
+# We need to build it a first time to extract sources translation files
+# in `i18` directory. react-intl-po need them to then compiled the translated messages
+# present in `translations/*.po` files.
+# Here we will build twice the front application but only the last one will be used.
```

Comment:
WDYT of the following alternative comment?

```Dockerfile
# We need to fake an empty translation file for the first front-end build that
# will extract sources translation files in the `i18n` directory. Then
# react-intl-po will use them to compile translated messages in
# `translations/*.po` files. Finally, the second front-end build bundles
# translations in the build that will be used in production.
```

Reply:
> ok